openmeetings-web/src/main/webapp/css/raw-room.css

```diff
@@ -425,11 +425,11 @@ html[dir="rtl"] .room-block .sb-wb .sidebar {
 .user-video .ui-dialog-titlebar
 , .sharer .ui-dialog-titlebar
 , .wb-tool-settings .ui-dialog-titlebar
-, .user-video .ui-dialog-titlebar .buttonpane
 {
 	background-color: var(--white);
 }
-.user-video .ui-dialog-titlebar .buttonpane {
+.user-video .buttonpane {
+	background-color: var(--white);
 	position: absolute;
```

Comment:
@sebawagner unfortunately this will NOT work :(
dialog has `position: absolute` so positioning of this buttonpane is not honored

Reply:
I fixed that before, let me have a look. It worked in chrome.

Reply:
It did work for me on Chrome.
I think there is some CSS missing that I did accidentally not commit. I think some top: 20px somewhere is missing.